src/main/java/application/rest/v1/ActionsEndpoint.java

```diff
@@ -203,50 +210,77 @@ public Response executeComponentCommand(String jsonstr, @Pattern(regexp = NAME_P
     @Produces(MediaType.APPLICATION_JSON)
     @Path("/commands")
     @Operation(
-            summary = "Retrieve the list of Kubernetes jobs for command actions, optionally filtered by user.",
+            summary = "Retrieve the list of Kubernetes jobs for command actions, optionally filtered by user and time and only jobs with completion timestamp newer than specified timestamp are returned",
             description = "Returns two lists: the list of Kubernetes jobs for command actions; the list of job actions for the jobs."
             )
     @APIResponses({@APIResponse(responseCode = "200", description = "OK"),
         @APIResponse(responseCode = "207", description = "Multi-Status (Error from Kubernetes API)"),
         @APIResponse(responseCode = "500", description = "Internal Server Error")})
-    public Response getCommands(@DefaultValue("") @QueryParam("user") @Parameter(description = "The user that submitted the command action") String user) {
+    public Response getCommands(@DefaultValue("") @QueryParam("user") @Parameter(description = "The user that submitted the command action") String user,
+                                @DefaultValue("") @QueryParam("time") @Parameter(description = "The job completion time stamp in millis since Jan 1, 1970 (yyyy-MM-dd'T'HH:mm:sss format) ") String time) {
         try {
             final ApiClient client = getApiClient();
             
             // Build the selector for the query.
             final Selector s = new Selector().addMatchLabel(KAPPNAV_JOB_TYPE, KAPPNAV_JOB_COMMAND_TYPE);
             if (user != null && !user.isEmpty()) {
                 s.addMatchLabel(KAPPNAV_JOB_USER_ID, user);
-            }
-            final String labelSelector = s.toString();
+            }           
             
+            // convert time to timestamp in millis yyyy-MM-dd'T'HH:mm:sss format
```

Comment:
delete word "millis" - input timestamp is time in "yyyy-MM-dd'T'HH:mm:sss" format.